File: src/rest-api-server/interfaces/MatchmakingQueues.ts
import MatchmakingUser from "./MatchmakingUser.ts";

export default interface MatchmakingQueues {
    matchmakingQueue25: MatchmakingUser[];
    matchmakingQueue50: MatchmakingUser[];
    matchmakingQueue100: MatchmakingUser[];
    matchmakingQueue200: MatchmakingUser[];
    matchmakingQueue500: MatchmakingUser[];
};

File: src/rest-api-server/src/main/java/com/licode/entities/users/MatchmakingQueues.java
package com.licode.entities.users;

import java.util.ArrayList;

public record MatchmakingQueues(
    ArrayList<MatchmakingUser> matchmakingQueue25,
    ArrayList<MatchmakingUser> matchmakingQueue50,
    ArrayList<MatchmakingUser> matchmakingQueue100,
    ArrayList<MatchmakingUser> matchmakingQueue200,
    ArrayList<MatchmakingUser> matchmakingQueue500) { }
